sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -298,7 +299,7 @@ def __init__(self):
 
 
 class ProxyConfiguration(object):
-    """Configurations for proxy.
+    """Configurations for a proxy.
```

Comment:
Configuration singular

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e